appcues/src/main/java/com/appcues/ui/primitive/ExperiencePrimitive.kt

```diff
@@ -89,6 +91,7 @@ private fun ExperiencePrimitive.getRole(): Role {
         is TextPrimitive -> Role.Button
         is TextInputPrimitive -> Role.Button
         is OptionSelectPrimitive -> Role.Button
+        is SpacerPrimitive -> Role.Image
```

Comment:
I think its Ok to mark as image? probably wont be used anyways. the only possible usecase where this would be clicable is if we actually make it have background or background image.